tests/network/hotplug.go

```diff
@@ -306,6 +306,14 @@ var _ = SIGDescribe("nic-hotunplug", func() {
 			By("verify unplugged interface is not reported in the VMI status")
 			vmi = verifyDynamicInterfaceChange(vmi, plugMethod)
 
+			By("verify unplugged iface cleared from VMI spec")
+			Eventually(func(g Gomega) {
```

Comment:
```
Eventually(func() {
				Expect(vmispec.LookupInterfaceByName(vmi.Spec.Domain.Devices.Interfaces, linuxBridgeNetworkName2)).To(BeNil(),
					"unplugged iface should be cleared from spec")
				Expect(lookupNetworkByName(vmi.Spec.Networks, linuxBridgeNetworkName2)).To(BeNil(),
					"unplugged iface corresponding network should be cleared from spec")
			}, 30*time.Second, 3*time.Second).Should(Succeed())
```
should work.

Reply:
The `Eventually` clouse will fail if one of the assertion fails, the combination of using `Gomega` object and `Succeed()` is what makes it works (in a way that if one assertion fail it is tolerated)
https://pkg.go.dev/github.com/onsi/gomega#Eventually (checkout Category 3)

Reply:
I didn't mean you should add the `Should(Succeed())`.
I meant the `g Gomega` is redundant.